agents/ten_packages/system/ten_ai_base/interface/ten_ai_base/const.py

```diff
@@ -3,15 +3,20 @@
 CMD_PROPERTY_TOOL = "tool"
 CMD_PROPERTY_RESULT = "tool_result"
 CMD_CHAT_COMPLETION_CALL = "chat_completion_call"
+CMD_GENERATE_IMAGE_CALL = "generate_image_call"
 CMD_IN_FLUSH = "flush"
 CMD_OUT_FLUSH = "flush"
 
 DATA_OUT_NAME = "text_data"
+RAW_DATA_OUT_NAME = "raw_text_data"
```

Comment:
How about `content_data` that has an extra `type` to indicate whether text or image buffer or something else?